src/hooks/voucherHooks.ts

```diff
@@ -35,12 +35,7 @@ export const useVoucher = (): {
   }, []);
 
   useEffect(() => {
-    if (isConnected && isMainserverUp && !isLoggedIn) {
-      logInCallback();
-    } else {
-      setIsLoading(false);
-      setIsError(true);
-    }
+    logInCallback();
   }, [isConnected, isMainserverUp, logInCallback]);
```

Comment:
do we still need both `isConnected, isMainserverUp` dependencies, even if we do not use its values?

Reply:
I think it would be correct to re-run the function in case of a disconnection. If any error still occurs, the `isError` statement will be `true`.

Reply:
ok. and therefore my question again, do we need both dependencies?

Reply:
Now I understand. Only 1 of them will be enough. I think `isConnected` will be enough and `loginCallback`

Reply:
https://github.com/smart-village-solutions/smart-village-app-app/pull/1105/commits/451e27e255b6e901bbf03e76d3dbe7a1b70cc1df